docs/running_psalm/issues/ComplexFunction.md

```diff
@@ -0,0 +1,3 @@
+# ComplexFunction
+
+Emitted when a function is too complicated for Psalm to analyze. Consider splitting it up to allow Psalm to analyze it.
```

Comment:
This could probably be expanded on, but I don't know enough about the conditions to in which this is emitted to document it correctly. If `Config::$max_graph_size` and `Config::$max_avg_path_length` ever become configurable that would also be good to mention here.